mantidproject/mantid
# MC Corrections workspace contains sample shape in name

### Description of work
This PR ensures the workspaces generated by the Calculate Monte Carlo Absorption tab of the Inelastic Corrections interface contains the sample shape in its name (just like is done for the Calculate PP tab).

Fixes #36597

### To test:
1. Open `Inelastic`->`Corrections` interface
2. Go to MC tab
3. Load iris26176_graphite002_red from the sample dataset
4. Choose Sample Height, Width and depth to be 0.1
5. The container front and back should be 0.01
6. The chemical formula for the Sample should be `O`
7. For the container, change the combo box to `Cross Sections`. Change all three cross sections to be 0.1
8. Click Run
9. See that the output workspaces contains the sample shape in its name (probably FlatPlate)

### Reviewer

Please comment on the points listed below ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)).
**Your comments will be used as part of the gatekeeper process, so please comment clearly on what you have checked during your review.** If changes are made to the PR during the review process then your final comment will be the most important for gatekeepers. In this comment you should make it clear why any earlier review is still valid, or confirm that all requested changes have been addressed.

#### Code Review

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Do the release notes conform to the [release notes guide](https://developer.mantidproject.org/Standards/ReleaseNotesGuide.html)?

#### Functional Tests

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### Gatekeeper

If you need to request changes to a PR then please add a comment and set the review status to "Request changes". This will stop the PR from showing up in the list for other gatekeepers.
